src/OpenRiaServices.Tools/Framework/AppDomainUtilities.cs

```diff
@@ -39,8 +39,10 @@ internal static void ConfigureAppDomain(ClientCodeGenerationOptions options)
             else
                 frameworkManifest = GetFrameworkManifest(options.ClientFrameworkPath);
 
-            AppDomain.CurrentDomain.ReflectionOnlyAssemblyResolve += AppDomainUtilities.ResolveFrameworkAssemblyVersioning;
+#if !NET6_0_OR_GREATER
```

Comment:
I wonder if it would make sense to start using "#if NETFRAMEWORK" instead.
What do you think ?